.github/workflows/default.yml

```diff
@@ -109,10 +109,17 @@ jobs:
         gradle: [ current ]
         java: [ 17 ]
         agp: ${{ fromJSON(needs.provide-agp-version-matrix.outputs.agp-versions) }}
+        exclude:
+          - gradle: current
+            java: 17
+            agp: 8.1.0-alpha01
         include:
           - gradle: 7.6
             java: 11
             agp: 4.2.2
+          - gradle: 8.0-rc-1
+            java: 17
+            agp: 8.1.0-alpha01
```

Comment:
I know it will start failing again when they release alpha-02, but I'm willing to pay that cost given how difficult the migration to AGP 8.0 will be. I want to be notified as soon as our workaround for https://issuetracker.google.com/issues/170650362 stops working